src/task/JoinAndReceiveIndexTask.ts

```diff
@@ -37,6 +37,7 @@ export default class JoinAndReceiveIndexTask extends BaseTask {
   async run(): Promise<void> {
     const indexFrame = await new Promise<SdkIndexFrame>((resolve, reject) => {
       const context = this.context;
+      let defaultVideoSubscriptionLimit: number = 25;
```

Comment:
Feel like this should just be set as default in `AudioVideoControllerState`? the possibility of it being 0 makes me a bit uncomfortable.

Reply:
Agree

Reply:
Yeah, I can have it set to default as 25 in the proto and the AudioVideoControllerState.